src/main/java/uk/gov/digital/ho/hocs/audit/application/RequestData.java

```diff
@@ -30,6 +31,7 @@ public boolean preHandle(HttpServletRequest request, HttpServletResponse respons
         MDC.put(USER_ID_HEADER, initialiseUserId(request));
         MDC.put(USERNAME_HEADER, initialiseUserName(request));
         MDC.put(USER_ROLES_HEADER, initialiseUserRoles(request));
+        MDC.put(GROUP_HEADER, initialiseGroups(request));
```

Comment:
I'm not too familiar with this so may need someone else to review, why do we need to do this stuff with groups?